services/user-service/models/user-model.ts

```diff
@@ -0,0 +1,44 @@
+// User.ts
```

Comment:
create a new file and name it as `base-model.ts`, it will be the model class that we going to extends onwards

```ts
import {
  InferAttributes,
  InferCreationAttributes,
  Model,
} from 'sequelize';


class BaseModel<
  T extends UnknownModel,
  ModelAttributes extends InferAttributes<T> = InferAttributes<T>,
  CreationAttributes extends
    InferCreationAttributes<T> = InferCreationAttributes<T>,
> extends Model<ModelAttributes, CreationAttributes> {
}

export default BaseModel
```